identity/did/bootstrapper.go

```diff
@@ -42,7 +46,17 @@ func (*Bootstrapper) Bootstrap(context map[string]interface{}) error {
 		return err
 	}
 
-	service := NewService(cfg, factoryContract, client)
+	txManager, ok := context[transactions.BootstrappedService].(transactions.Manager)
+	if !ok {
+		return errors.New("transactions repository not initialised")
+	}
+
+	if _, ok := context[bootstrap.BootstrappedQueueServer]; !ok {
```

Comment:
Can you use the same format as above?
```suggestion
      queueSrv, ok := context[bootstrap.BootstrappedQueueServer].(*queue.Server)
      if !ok {
         return errors.New("queue hasn't been initialized")
      }
```